.github/workflows/slither.yml

```diff
@@ -0,0 +1,45 @@
+name: Slither Analysis
+
+on:
+  push:
+    branches: [develop, master]
+  pull_request:
+    branches: [develop]
```

Comment:
why `master` is omitted?

Reply:
Because the report will be generated on push to develop